.gitignore

```diff
@@ -4,3 +4,4 @@ npm-debug.log
 lerna-debug.log
 yarn-error.log
 package-lock.json
+.vscode/
```

Comment:
Since this is IDE specific and not something related to the library I'd rather include this in the global `.gitignore`. See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c.